FFMpegCore/FFMPEG/Argument/ArgumentContainer.cs

```diff
@@ -117,6 +116,16 @@ public bool ContainsKey(Type key)
             return _args.ContainsKey(key);
         }
 
+        public int CountExistedKeys(params Type[] types)
```

Comment:
Suggestion:
Change this method to
```csharp
public bool ContainsOnlyOneOf(params Type[] types)
{
  return types.Count(t => _args.ContainsKey(t)) == 1;
}
```
To make it shorter and simpler

Reply:
Sure